pyproject.toml

```diff
@@ -50,7 +50,7 @@ before-build = [
 
 # By default there is no repair command on windows set, use delvewheel
 repair-wheel-command = [
-    "pip install delvewheel==0.0.18",
+    "pip install delvewheel==1.6.0",
```

Comment:
`0.0.18` to `1.6.0` ... I think I should try to understand what `delvewheel` does before merging this 🤔

Reply:
`Delvewheel` is a command-line tool that modifies Python wheel packages and injects dependency dynamic libraries to make them independently bundled.

I built the Windows x64 version on my GitHub pipeline and it looks fine.
https://github.com/LY1806620741/unitree_sdk2_python/blob/37967b640de394992fc7bf06147daf856213a38b/.github/workflows/python-package.yml#L48

In the end, it injected a complete DLL file. see https://github.com/LY1806620741/unitree_sdk2_python/releases/tag/3.11

well, `delvewheel` 0.0.18 supports up to Python 3.9.see https://pypi.org/project/delvewheel/0.0.18/

at 1.0.0,it Label project as production/stable. https://github.com/adang1345/delvewheel/blob/master/CHANGELOG.md